app/web/features/team/Team.tsx

```diff
@@ -72,43 +72,57 @@ export default function Team() {
           justifyContent="center"
           alignItems="stretch"
         >
-          {TeamData.map(({ name, founder, role, location, img, link }) => (
-            <Grid key={name} item xs={12} md={!!founder ? 5 : 4}>
-              <Card elevation={founder ? 3 : 1} className={classes.cardWrapper}>
-                <CardContent className={classes.card}>
-                  <MuiAvatar
-                    alt={`Headshot of ${name}`}
-                    src={img}
-                    className={classes.avatar}
-                  />
-                  <div className={classes.cardContent}>
-                    <Typography variant={founder ? "h1" : "h2"} component="h2">
-                      {name}
-                    </Typography>
-                    {founder && (
-                      <Typography variant="h2" component="h3">
-                        Co-founder
+          {TeamData.map(
+            ({ name, director, board_position, role, location, img, link }) => (
+              <Grid key={name} item xs={12} md={!!director ? 4 : 4}>
+                <Card
+                  elevation={director ? 3 : 1}
```

Comment:
Get rid of the elevation difference and just have their own section for directors, Ireckon